docs/diagrams/ModelAddressBookClassDiagram.puml

```diff
@@ -0,0 +1,45 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+Class ModelManager
+Class Appointment
+
+Package "Address Book Models" as Model <<Rectangle>>{
+Class AddressBook
+
+Class UniquePersonList
+Class Person
+Class Address
+Class Email
+Class Name
+Class Phone
+
+Class UniqueTagList
+Class Tag
+Class "  Name  "
+}
+
+Class HiddenOutside #FFFFFF
+
+
+AddressBook *--> "1" UniquePersonList
+AddressBook *--> "1" UniqueTagList
+UniquePersonList --> "~* all" Person
+UniqueTagList --> "~*" Tag
+Person *--> Name
+Person *--> Phone
+Person *--> Email
+Person *--> Address
+Person *--> "*" Tag
+Tag *--> "1" "  Name  "
+
+Name -[hidden]right-> Phone
+Phone -[hidden]right-> Address
+Address -[hidden]right-> Email
+
+ModelManager -->"~* filtered" Person
+Appointment -> Person
```

Comment:
Are we missing
`ModelManager --> "~* filtered" Tag` as `ModelManager` also keeps a filtered list of tags?

Reply:
You are right, fixed in the latest commit 0c95718.